apps/frontend/src/libs/components/switch/switch.tsx

```diff
@@ -0,0 +1,44 @@
+import { Button } from "~/libs/components/components.js";
+import { type WheelEditMode } from "~/libs/types/types.js";
+
+import styles from "./styles.module.css";
+
+type Properties = {
+	currentMode: WheelEditMode;
+	leftButtonProperties: SwitchButtonProperties;
+	onToggleMode: () => void;
+	rightButtonProperties: SwitchButtonProperties;
+};
+
+type SwitchButtonProperties = {
+	label: string;
+	mode: WheelEditMode;
```

Comment:
I think it would be better to use generic here, to reuse this component with different mode types